Data/pickle_tags.py

```diff
@@ -0,0 +1,47 @@
+import csv
```

Comment:
Originally it was for writing tags to a pickled file, but now that we changed the functionality of this file I renamed it.